.docs/contributing-book/src/documentation/project-quality/documentation/code/style.md

```diff
@@ -46,6 +46,28 @@ The general approach is to omit a type if the compiler does not throw an error h
 {{#include ../../../../code/style_guide/src/lib.sw:type_annotation}}
 ```
 
+## Field initialization shorthand
```

Comment:
```suggestion
## Field Initialization Shorthand
```
I believe the other headings in the file are also capitalized

Reply:
I sometimes wonder which part to capitalize for emphasis or if everything should be done.

Reply:
What I generally follow is [title capitalization](https://www.grammarly.com/blog/capitalization-in-the-titles/#:~:text=According%20to%20most%20style%20guides,part%20of%20speech%20they%20are.) rules for markdown headers.